src/ContentMessages.js

```diff
@@ -513,7 +526,7 @@ export default class ContentMessages {
     }
 
     getCurrentUploads() {
-        return this.inprogress;
+        return this.inprogress.filter(u => !u.canceled);
```

Comment:
Hmm, we seem to have 2 spellings of `cancelled` / `canceled` in the PR... Can we pick just one instead?

Reply:
ah yes